snapcraft/internal/cache/_file.py

```diff
@@ -26,12 +26,12 @@
 class FileCache(SnapcraftCache):
     """Generic file cache."""
 
-    def __init__(self):
+    def __init__(self, *, taxonomy: str = "files") -> None:
```

Comment:
Github decided to hide my comment, so I'll try again:

I don't know what to think when I see "taxonomy" as an argument. It's a field of study. From how it's used, perhaps "namespace" would make more sense?

Reply:
I was having troubles thinking of a word. Your suggestion is so obvious now!